schemas/view-schema.json

```diff
@@ -125,6 +125,11 @@
               "description": "Whether the View corresponding to this event is considered active",
               "readOnly": true
             },
+            "is_slow_rendered": {
+              "type": "boolean",
+              "description": "Whether the View had a low average refresh rate",
```

Comment:
how do we define _low_? we can mention it here to avoid confusion

Reply:
Low is defined in the related RFC (an average framerate < 55 fps). But like `frozen_frame` we don't need to have the exact number is this may be changed later.

Reply:
Is it android (or mobile ?) specific ?
If the threshold can change later, should we not, store the `average_framerate` instead and compute the boolean in the backend or even at the query time with `average_framerate < 55` ?

Reply:
It's mobile specific indeed, the definition of how we compute it might change and be linked to data that's not part of the `average_framerate`. It's easier to let the SDK compute it.